app/js/features/escrow/saga.js

```diff
@@ -120,7 +121,55 @@ export function *onLoadEscrows() {
   yield takeEvery(LOAD_ESCROWS, doLoadEscrows);
 }
 
+
+export function *checkUserRating({address}) {
+  try {
+    address = address || web3.eth.defaultAccount;
+    const state = yield select();
+    const offers = Object.values(state.metadata.offers).filter(offer => offer.owner.toLowerCase() === address.toLowerCase());
```

Comment:
all address on the store should be lowerCase.
we do this pre-processing when we store them in the reducer